yt_dlp/extractor/altcensored.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+from .common import InfoExtractor
+from .archiveorg import ArchiveOrgIE
+from ..utils import (
+    int_or_none,
+    orderedSet,
+    urljoin,
+    InAdvancePagedList,
+)
+
+
+class AltCensoredIE(InfoExtractor):
+    IE_NAME = 'altcensored'
+    _VALID_URL = r'https?://(?:www\.)?altcensored\.com/(?:watch\?v=|embed/)(?P<id>[^/?#]+)'
+    _TESTS = [{
+        'url': 'https://www.altcensored.com/watch?v=k0srjLSkga8',
+        'info_dict': {
+            "id": "youtube-k0srjLSkga8",
+            "ext": "webm",
+            "title": "QUELLES SONT LES CONSÉQUENCES DE L'HYPERSEXUALISATION DE LA SOCIÉTÉ ?",
```

Comment:
Use single quotes where it makes sense
```suggestion
            'id': 'youtube-k0srjLSkga8',
            'ext': 'webm',
            'title': "QUELLES SONT LES CONSÉQUENCES DE L'HYPERSEXUALISATION DE LA SOCIÉTÉ ?",
```